### PR TITLE
5 Column Footer (only NL first)

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
This adds an extra column in the footer of the NL website, so we could cluster pages with a theme or topic which is not a product, service or an about us page. This feels more natural to me. For example pages about Convenant Medische Technologie, 70:20:10 and Performance Support. Tweaked some other things too. 

- [x] Add 5th column on dutch footer
- [x] Make the footer a bit more wide
- [x] Make link :hover `$orange` instead of white 
- [x] Replace all color literals with variables and fix other scss-linter stuff
- [x] Add search
- [x] Name the about us column `About Us` instead of `Defacto`
- [x] State the website is open source in the copyright frase and link to this repo 

At this moment we only have 1 theme page, but I am trying to release other themed landing pages really soon, because we want to be found at trending topics. 
That's why I put an extra example link in the screenshot below: 

## Screenshot: 

footer-impression

### CODE REVIEW

File: locales/nl.yml [hunk 1]
```diff
@@ -123,10 +123,13 @@ nl:
     text_md: "Dit betekent niet dat je iets verkeerd hebt gedaan, het kan zijn dat we de pagina hernoemd of verplaatst hebben."
   footer:
     capp: "CAPP Leren & Ontwikkelen"
-    copyright_html: "© 1987 - %{year} Defacto"
+    copyright_html: "Deze website is %{link} © 1987 - %{year} Defacto"
```
Comment (sn3p) [inline locales/nl.yml]: ❤️

File: source/stylesheets/_footer.scss [hunk 2]
```diff
@@ -21,12 +35,18 @@
     }
 
     @include media($large-up) {
+      // Only NL has 5 columns yet
+      @at-root body.nl .footer .col {
+        @include span-columns(2.4);
```
Comment (sn3p) [inline source/stylesheets/_footer.scss]: Doe we need `@at-root` here?
Doesn't this work?
```scss
@include media($large-up) {
  body.nl & {
    @include span-columns(2.4);
  }
}
```

Comment (RonaldDijkstra) [inline source/stylesheets/_footer.scss]: It does.

File: locales/nl.yml (hunk 1 above)
Comment (RonaldDijkstra) [inline locales/nl.yml]: 💯

**Pull request merged**